sites/svelte.dev/src/routes/_components/Demo.svelte

```diff
@@ -32,7 +32,7 @@
 </script>
 
 <Section --background="var(--sk-back-2)">
-	<h3>Why Svelte?</h3>
+	<h3>build with elegance</h3>
```

Comment:
'build with elegance' reads weirdly to me, i don't think of elegance is being something you build with. maybe something like this?

```suggestion
	<h3>build with ease</h3>
```